docs/advanced-configurations.md

```diff
@@ -0,0 +1,108 @@
+# Advanced Configurations
+
+The NowSecure Action enables a number of customizations and features. These are
+documented here.
+
+- [Action Configuration](#action-configuration) (recommended reading)
+- [GitHub Summary Page](#github-summary-page)
+- [Per-app Filters](#per-app-filters)
+- [Configuring Finding IDs Generation](#configuring-finding-ids-generation)
+- [SBOM Generation](#sbom-generation)
+
+## Action Configuration
+
+The NowSecure Action is further configurable via an additional file stored in the repo.
+By default, it is called `.nsconfig.yml` and is placed in the root of the repo, however the name and path are configurable via the `config` and `config_path` workflow parameters.
+
+This file allows you to configure:
+
+- A minimum severity filter (the default includes critical, high and medium findings)
+- A list of check IDs to include
+- A list of check IDs to exclude
+- Whether warnings should be shown
+
```

Comment:
See a list of check IDs in NowSecure HelpCenter: https://support.nowsecure.com/hc/en-us/articles/10395931666445